daemon/logger/splunk/splunk.go

```diff
@@ -363,6 +364,11 @@ func (l *splunkLoggerJSON) Log(msg *logger.Message) error {
 }
 
 func (l *splunkLoggerRaw) Log(msg *logger.Message) error {
+	// empty or whitespace-only messages are not accepted by HEC
+	if strings.TrimSpace(string(msg.Line)) == "" {
```

Comment:
one more thing I was thinking, as performance may be important for logging; should we check for real empty first? e.g.;

```go
if len(msg.Line) == 0 || strings.TrimSpace(....
```

(sorry, I'm traveling, reading all notifications from my phone :sweat_smile:)

Reply:
@thaJeztah do you have further feedback for me? And how can we get this merged soon?